.pre-commit-config.yaml

```diff
@@ -35,5 +29,5 @@ ci:
   autofix_prs: true
   autoupdate_commit_msg: "[pre-commit.ci] pre-commit autoupdate"
   autoupdate_schedule: weekly
-  skip: [black,isort]
+  skip: [black]
```

Comment:
How about `autofix_prs` set to `false` and we don't skip anything?

Reply:
yeah we said we weren't gonna do autofix at the start anyway

Reply:
Changed as of 1239a4b2